### PR TITLE
match another domain, also fixed a minor issue

### PR DESCRIPTION
**Pull request by Arthraim**

### CODE REVIEW

File: build/manifest.json [hunk 1]
```diff
@@ -22,8 +22,10 @@
   },  
   "content_scripts": [
     {
-      "matches": [
-        "https://m.okjike.com/*"
+      "matches": [ "*://*/*" ],
+      "include_globs": [
+        "*://m.okjike.com/*",
+        "*://m.ruguoapp.com/*"
```
Comment (Arthraim) [inline build/manifest.json]: didn't really test if this change works, can you double check it?

File: greasemonkey.meta.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// ==UserScript==
+// @name         To Jike
+// @namespace    http://lab.duoie.cc/
+// @version      1.2
+// @description  Go to web jike from share page.
+// @author       未枝丫
+// @copyright    2018, soyaine (https://github.com/soyaine)
+// @license MIT
+// @grant        none
+// @include     /^https?://m\.(okjike|ruguoapp)\.com/.*$/
```
Comment (Arthraim) [inline greasemonkey.meta.js]: match more domains here as well

**Comment by soyaine** [comment]
Thank you, but it seems that this feature has been supported officially.

**Comment by Arthraim** [comment]
Indeed -.- nvm then

**Pull request closed without merging**